framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1257,7 +1259,7 @@ private boolean checkSuppressWarnings(String /*@Nullable*/ [] userSwKeys, String
      *         it is contained by a declaration with an appropriately-valued
      *         {@literal @}SuppressWarnings annotation; false otherwise
      */
-    private boolean shouldSuppressWarnings(Tree tree, String errKey) {
+    public boolean shouldSuppressWarnings(Tree tree, String errKey) {
```

Comment:
I think the new method would be identical, having the exact same number of lines. Only three lines would be different. Therefore I think keeping this one public is better than adding code duplication. Please let me know if you agree.